ivy/src/main/java/sbt/mavenint/SbtPomExtraProperties.java

```diff
@@ -12,7 +12,11 @@
     public static final String POM_INFO_KEY_PREFIX = "info.";
     public static final String POM_SCALA_VERSION = "scalaVersion";
     public static final String POM_SBT_VERSION = "sbtVersion";
+    /** @deprecated Use {@link #POM_SCALA_API_KEY} instead */
+    @Deprecated
```

Comment:
Well, to resolve this, here's what I'd say:

1. We do a survey of recently released scala libs and figure out the % which are actively using the apiURL feature.  Given its low press, I'm assuming it's not heavily used yet.
2. If we find apiURL isn't heavily used (and given its recent addition) we deprecate it in favor of scalaApiURL and javaApiUrl. Basically, if people have trouble seeing the new libs, they should be freely able to upgrade to sbt 0.13.8, since it's bin-compat.   We need to make sure this is always true.
3. After this fix, we can start advertising the feature more, possibly recommending the OSS community us it.   I'll let you, @indrajitr, take the lead on pushing its use and ensuring libraries across teh OSS ecosystem can cross-linke docs.

Sound likea  good plan?